### ISSUE TITLE
Image sprite is too big in size 

### ISSUE THREAD

**Issue by amitpatil321** [contentco/quill-emoji#160]
After bundling my react project, I noticed its size is huge and after analyzing the build I found that quill-emoji sprite is huge in size (almost 7mb).

After compression image become 1.7MB only. 
If you want I can create pull request with that image.

**Comment by ginobean** [comment]
Agreed. This is needed. Also, of the list of emotions, it would nice to be able to pick a subset of the 
emoticons (e.g. just the emoticon faces), thus reducing the network/load times even more.
